tests/test_upnp_client.py

```diff
@@ -101,6 +101,20 @@ async def test_init_bad_xml(self) -> None:
         with pytest.raises(UpnpXmlContentError):
             await factory.async_create_device("http://dlna_dmr:1234/device.xml")
 
+    @pytest.mark.asyncio
+    async def test_empty_descriptor(self) -> None:
+        """Test device with an empty descriptor file called in description.xml."""
+        responses = dict(RESPONSE_MAP)
+        responses[("GET", "http://dlna_dmr:1234/device.xml")] = (
+            200,
+            {},
+            read_file("dlna/dmr/device_with_empty_descriptor.xml"),
+        )
+        requester = UpnpTestRequester(responses)
+        factory = UpnpFactory(requester)
+        with pytest.raises(UpnpXmlContentError):
```

Comment:
This should be expecting `UpnpXmlParseError`, since that's what `_async_create_service` is re-raising.